Command/SonataDoctrineUtilsCommand.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/*
+ * This file is part of the Sonata Connector project.
+ *
+ * (c) Sylvain Rascar <sylvain.rascar@fullsix.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Command;
+
+use Gaufrette\Filesystem;
+use Gaufrette\Adapter\Local as LocalAdapter;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Doctrine\ORM\Mapping\ClassMetadata;
+
+/**
+ * Return useful data on the database schema
+ *
+ * Class SonataDoctrineUtils
+ * @package Sonata\ProductBundle\Command
+ */
+class SonataDoctrineUtilsCommand extends ContainerAwareCommand
+{
+    /**
+     * @var string
+     */
+    protected $defaultDumpDirectory;
+
+    /**
+     * @var array $metadata
+     */
+    protected $metadata;
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('sonata:doctrine:utils')
```

Comment:
You should rename this command, it does not speak for what it does. Something like ``sonata:core:dump-doctrine-metadata`` is more self-explainable.